blocks.md

```diff
@@ -0,0 +1,111 @@
+
+<h3>Block Plugins and the Block Directory</h3>
+
+The Block Directory contains only Block Plugins, that is to say plugins that contain only a single independent block and a minimum of supporting code. Block plugins have the following characteristics:
+
+1. A specific type of WordPress plugin, with the same structure including a readme.txt file.
+1. Containing only blocks (typically one).
+1. Contain a minimum of server-side code. 
+1. Do not have any UI outside of the post editor.
+
+In addition to following the guidelines that apply to all WordPress plugins, Block Plugins that are submitted to the Block Directory must adhere to these guidelines:
+
+<h3>Guidelines</h3>
+
+<h4>1. Block Plugins are for the Block Editor.</h4>
+
+Your block plugins should contain a block, and a minimum of other supporting code. It should not contain any UX outside of the editor, such as WordPress options or `wp-admin` menus. Server-side code should be kept to a minimum.
+
+While plugins that provide styles for or extend existing core blocks are permitted, please be aware that they are not yet supported by the Block Editor's seamless install, which means they cannot be included in the Block Directory just yet.
+
+<h4>2. Block Plugins are separate blocks.</h4>
+
+Block plugins are intended to be single purpose, separate, independent blocks, not bundles or compendiums of many blocks. In most cases, a Block Plugin should contain only one single block. It should not contain multiple blocks that could reasonably be split up into separate, independent plugins.
+
+Block plugins may contain more than one block where a clearly necessary parent/child or container/content dependency exists; for example a list block that contains list item blocks.
+
+<h4>3. Plugin and block names should reflect the block's purpose.</h4>
+
+Plugin titles and block titles should describe what the block does in a way that helps users easily understand its purpose. In most cases the plugin title and the block title should be identical or very similar. Some examples of good plugin and block names include:
+
+`Rainbow Block`
+`Sepia Image Grid`
+`Business Hours Block`
+
+Please avoid plugin and block titles unrelated to the block itself, or that cannot be easily distinguished from core blocks. Some examples of unhelpful plugin and block names are things such as:
+
+`PluginCo Inc Block`
+`Widget`
+`Image Block`
+
+You must refrain from names and titles that include trademarks or names that you do not legally represent; for example you cannot name a block "Facerange Block" unless you work for Facerange and have their permission.
```

Comment:
"The same trademark restrictions for plugin titles exist for block titles. This means that a block may not be named "Facerange Block" unless developed by a legal representative of Facerange. For example, if a block is meant to show custom discount information for an ecommerce plugin, then a good name would be "Custom Discounts for WeeCommerce."

We don't use 'you' in other places, so let's try to be consistent :) Tone helps translators!

Reply:
I've added the first sentence. The second seems to suggest that a block that depends on WeeCommerce would be permitted - which currently it would not.